### PR TITLE
:truck: update: Add 119th CatchUp Summary

### PR DESCRIPTION
**Pull request by tusharnankani**
This is a ~god-awful~ draft PR.

Please review.

### CODE REVIEW

File: summary/sessions/119/attendees.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+==== Attendees
+
+. Abhishek Vallecha
+. Aditya Oberai
+. Anil Gohan
+. Chirag Lulla
+. Darshit Suratwala
+. Dheeraj Lalwani
```
Comment (SirusCodes) [inline summary/sessions/119/attendees.adoc:8]: Maybe you can write a script😅

Comment (tusharnankani) [reply summary/sessions/119/attendees.adoc:8]: Yes, @dheerajdlalwani had an interesting idea. He's giving it a shot.

File: summary/sessions/119/content.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    ** The project was implemented using AWS Lambda Validate, Firebase Functions and the interface was built using Vue.js. 
+    ** We discussed interesting problems around it like how scaling such applications is not a problem, but abuse of such services becomes a problem. 
+    ** Relevant Links
+        *** link:https://vadosware.io/post/its-never-been-easier-or-harder-to-self-host-email[Self-Hosted email is the hardest it's ever been, but also the easiest.^]
+        *** link:https://cfenollosa.com/blog/after-self-hosting-my-email-for-twenty-three-years-i-have-thrown-in-the-towel-the-oligopoly-has-won.html[After self-hosting my email for twenty-three years I have thrown in the towel. The oligopoly has won.^]
+        *** link:https://explained-from-first-principles.com/email[Email - explained from first principles^]
+* Harsh Kapadia asked why people use Public and Private Keys and what the usage is. link:https://alexcabal.com/alex-cabal-gpg-public-key.asc[Example of a GPG Public Key^].
```
Comment (SirusCodes) [inline summary/sessions/119/content.adoc:20]: Where? TLS? SSH?

**Review by SirusCodes — CHANGES_REQUESTED** [review]
Rest LGTM

File: summary/sessions/119/content.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+==== Topics Discussed
+
+* Saifuddin Saifee asked link:https://support.google.com/youtube/answer/12827017[Ambient Mode in YouTube^] works on the backend.
+    ** Most of us presumed the feature would be handled on the fly, on the client side — by picking up a few primary colours from the video.
+    ** However, Ramyak said colours are pre-processed alongside other metadata when a video is being uploaded. 
+* We held our second edition of link:https://talks.ourtech.community[OTC Talks^] session with link:https://twitter.com/omtalk[Omkar Khair^] giving a talk on 'Internals of an Email Relay'. 
+    ** Find Abstract and other relevant links — link:https://talks.ourtech.community/2[OTC Talks #2^]
+    ** The talk covered interesting ideas about Email Relay, Email Meta Data, Signing Keys, Zero Ops approach, and the total cost of building one.
+    ** Omkar also demonstrated the application link:https://dopemin.com[Dopemin - Firewall for your Email^] — that he along with 8 users (friends and family) have been using seamlessly, for the past three years.
+    ** The project was implemented using AWS Lambda Validate, Firebase Functions and the interface was built using Vue.js.
```
Comment (PranavDani) [inline summary/sessions/119/content.adoc:14]:
```suggestion
    ** The project was implemented using AWS Lambda Validate, Firebase Functions, and the interface was built using link:https://vuejs.org[Vue.js^]. 
```

File: summary/sessions/119/content.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Date: 18-02-2023
+
+Duration: 3 hrs 30 mins
+
+==== Topics Discussed
+
+* Saifuddin Saifee asked link:https://support.google.com/youtube/answer/12827017[Ambient Mode in YouTube^] works on the backend.
+    ** Most of us presumed the feature would be handled on the fly, on the client side — by picking up a few primary colours from the video.
```
Comment (PranavDani) [inline summary/sessions/119/content.adoc:8]:
```suggestion
    ** Most of us presumed that the feature would be handled on the fly, on the client side — by picking up a few primary colors from the video.
```

File: summary/sessions/119/attendees.adoc (hunk 1 above)
Comment (HarshKapadia2) [reply summary/sessions/119/attendees.adoc:8]: Are we waiting for that to merge this?

**Comment by HarshKapadia2** [comment]
@tusharnankani I've made the changes that I thought were required. Feel free to change things up. Thank you for the PR and sorry for the delay in adding more details!

Also, please remember to always leave one line blank at the end of files. It is a good practice. This is usually because [different OSs treat line endings differently](https://dev.harshkapadia.me/resources#the-missing-semester-of-cs-education) and we want to avoid any issues. Git frequently gives warnings about this stuff on running `git add`. Git has a setting to modify line endings as well!

**Comment by tusharnankani** [comment]
> @tusharnankani I've made the changes that I thought were required. Feel free to change things up. Thank you for the PR and sorry for the delay in adding more details!

Thank you so much, I was waiting for just these additions. 

> Also, please remember to always leave one line blank at the end of files. It is a good practice. This is usually because [different OSs treat line endings differently](https://dev.harshkapadia.me/resources#the-missing-semester-of-cs-education) and we want to avoid any issues. Git frequently gives warnings about this stuff on running `git add`. Git has a setting to modify line endings as well!

Okay, understood. Makes sense!

File: summary/sessions/119/attendees.adoc (hunk 1 above)
Comment (tusharnankani) [reply summary/sessions/119/attendees.adoc]: For now, I tested the script mentioned in #139 and updated the attendees.adoc

Merging now!

**Pull request merged**